docs/cli_tools.rst

```diff
@@ -0,0 +1,26 @@
+==================
+Command-line Tools
+==================
+
+Pinout
+======
+
+The gpiozero package contains a database of information about the various
+revisions of Raspberry Pi. This is queried by the ``pinout`` command-line
+tool to write details of the GPIO pins available.
```

Comment:
My thought is that "display" could be misleading. How about "output"?

Depends on the audience. Thinking of my own children, for end of KS2 "output" seems a good choice, but earlier I'd stick to "write"/"display" or, maybe, "give".